docs/views/views.mdx

```diff
@@ -1,104 +1,7 @@
-import CodeBlock from "@theme/CodeBlock";
-
-# Instance views
+# Instance Views
 
 Zeno Instance Views are modular renderers for different data types and tasks.
```

Comment:
Instance Views let you specify how Zeno should render your data. 
There are two ways to define instance views: